styx-common/src/main/java/com/spotify/styx/util/MissingImportantPropertyException.java

```diff
@@ -0,0 +1,33 @@
+/*-
+ * -\-\-
+ * Spotify Styx Common
+ * --
+ * Copyright (C) 2016 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+
+package com.spotify.styx.util;
+
+import java.io.IOException;
+
+/**
+ * An exception thrown when an important property is missing in a resource.
+ */
+public class MissingImportantPropertyException extends IOException {
```

Comment:
Also, how `important` does not strictly imply required. How about calling it `MissingRequiredPropertyException` ?